llvm/lib/Target/AMDGPU/SIFoldOperands.cpp

```diff
@@ -2760,9 +2868,11 @@ bool SIFoldOperandsImpl::run(MachineFunction &MF) {
         continue;
       }
 
-      if (MI.isRegSequence() && tryFoldRegSequence(MI)) {
-        Changed = true;
-        continue;
+      if (MI.isRegSequence()) {
```

Comment:
```suggestion
      if (MI.isRegSequence())
```
nit: can we save a couple of braces?